analyticsdataserver/settings/base.py

```diff
@@ -315,8 +316,6 @@
 ANALYTICS_DATABASE = 'default'
 DATABASE_ROUTERS = ['analyticsdataserver.router.AnalyticsApiRouter']
 
-ENABLE_ADMIN_SITE = False
```

Comment:
@mulby Any idea about why admin site is disabled in production? We need to enable this now and wanted to have your opinion.

Reply:
I don't remember, sorry. My guess is that we felt like it wasn't worth the security risk if we didn't have a need for it.